examples/pytorch/BertNewsClassification/README.md

```diff
@@ -0,0 +1,75 @@
+## Bert Sentiment Analysis example with MLFlow
+
+In this example, we have used a pretrained Bert model to classify the news reports into any one of the 4 different categories("world", "Sports", "Business", "Sci/Tech").
+The autolog code uses Pytorch Lightning's MLFlowLogger to log metrics.
+The code is trained using pytorch lightning loop and the autolog function call in the main - `autolog()`
+is responsible for logging the params, metrics, model summary and the model.
+ 
+
+### Code related to MLflow:
+* [`mlflow.pytorch.pytorch_autolog`]
+This is an experimental api that logs ML model artifacts and metrics.
+The metrics are logged during training of the model.
+
+## Setup
+
+Mlflow tracking UI is set to `http://localhost:5000` with experiment name as `Default` in the script - `bert_classification.py`
```

Comment:
Note that one advantage of not having tracking URI as a parameter to the project / just relying on the MLflow default URI is that users don't need to launch a tracking server at http://localhost:5000/ via  ``mlflow server`` in order to run the example script

Reply:
@smurching The biggest benefit of MLflow esp in a team environment is for people to be able share their models / experiment runs. So using a common tracking server that has been launched separately is going to be an important scenario. For user's that are just testing on their local machines it is ok to launch separately each time. If MLFLOW_TRACKING_URI environment variable is the common way people use MLflow shared tracking server link, then we can make this change. Current document pages do not clearly describe this part

Reply:
will address this in https://github.com/mlflow/mlflow/pull/3608